client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -107,7 +107,7 @@ class SharingButtons extends Component {
 					initialized={ !! buttons && !! settings }
 					saving={ isSaving } />
 				<ButtonsOptions
-					postTypes={ Object.values( postTypes ) }
+					postTypes={ postTypes }
```

Comment:
Ha! I spent a good 15 minutes going back and forth on that actually. Yes, let's do it!